src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -94,15 +92,13 @@ export const Map: React.FC<MapProps> = ({
     (gridPosition: GridPosition) => {
       if (!userUid) return;
 
-      setLocationData({ userId: userUid, locationName: venueName });
```

Comment:
Why is this not needed anymore? Is it a duplicate?

Reply:
No, it's legacy logic

Reply:
> Why was this needed beforehand and not needed now?

We needed it when we had grid on partymap. But we have never used it